cmd/dependency-order/main.go

```diff
@@ -27,19 +25,9 @@ func main() {
 		log.Fatalf("failed to walk directory: %v", err)
 	}
 
-	var includedPlugins []*plugin.Plugin
-	var err error
-	if pluginsMatch := os.Getenv("PLUGINS"); pluginsMatch != "" {
-		includedPlugins, err = plugin.FilterByPluginsEnv(plugins, pluginsMatch)
-		if err != nil {
-			log.Fatalf("failed to filter plugins by PLUGINS env var: %v", err)
-		}
-	} else {
-		// Filter by changed plugins (for PR builds)
-		includedPlugins, err = plugin.FilterByChangedFiles(plugins, envconfig.OsLookuper())
-		if err != nil {
-			log.Fatalf("failed to filter plugins by changed files: %v", err)
-		}
+	includedPlugins, err := plugin.FilterByPluginsEnv(plugins, os.Getenv("PLUGINS"))
```

Comment:
Now all commands other than changed-plugins only look at one source of information instead of duplicating this logic across them all.